src/components/BigText/index.module.css

```diff
@@ -1,8 +1,13 @@
 @import "../../styles/settings.css";
 
 .root {
-  font-size: $size-26;
-  line-height: $size-36;
+  font-size: $size-16;
+  line-height: $size-20;
+
+  @media (min-width: 700px) {
```

Comment:
We have `postcss-preset-env`, we can use the proposed media queries range like https://preset-env.cssdb.org/features#media-query-ranges

Another thing is I think we should implement styles desktop first right now. The first design will always be desktop usually so it might be a bit confusing for people to pickup the project and see that styles are being applied mobile first. I know that from a performance standpoint it might not be the best, but we already know João is probably going to go desktop first on his designs, so best to prevent it that way.

Reply:
Also, better to keep that `700px` on a variable on `settings`